bootstrap.go

```diff
@@ -257,23 +256,24 @@ func checkBootstrapped(s Session) (bool, error) {
 
 // getTiDBVar gets variable value from mysql.tidb table.
 // Those variables are used by TiDB server.
-func getTiDBVar(s Session, name string) (types.Datum, error) {
+func getTiDBVar(s Session, name string) (string, bool, error) {
```

Comment:
use `Named return values` is more readable, or add some comment about return values.